etc/tasks/deleteEtherpadSessions.js

```diff
@@ -0,0 +1,78 @@
+/*!
+ * Copyright 2015 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+var _ = require('underscore');
+var util = require('util');
+
+var Cassandra = require('oae-util/lib/cassandra');
+var log = require('oae-logger').logger('delete-etherpad-sessions');
+var OAE = require('oae-util/lib/oae');
+
+// The application configuration
+var config = require('../../config').config;
```

Comment:
Should we keep into account that a custom config file can be specified in app.js ?